ng/views/createPost.html

```diff
@@ -1,98 +1,84 @@
+                    <fieldset ng-disabled="isSubmitting">
+                        <input 
+                            ng-model-options="{updateOn: 'blur'}"
+                            ng-model="newPosting.price.minimum" 
+                            type="text" 
+                            class="form-control"
+                            placeholder="Minimum Price"
+                            name="postLowPriceName"
+                            id="inputPriceLow">
+                        <input 
+                            ng-model-options="{updateOn: 'blur'}"
+                            ng-model="newPosting.price.maximum" 
+                            type="text" 
+                            class="form-control" 
+                            placeholder="Maximum Price"
+                            name="postHighPriceName"
+                            id="inputPriceHigh">
+                    </fieldset>
+                    <div ng-repeat="message in priceErrorMessages">
+                        <h4 style="color: red">{{message}}</h4>
                     </div>
-                    <div>
-                        <img style="width: 30%; margin: 1% 0;" src="../images/addImage.png" alt=""> 
+                </div>
+                <div class="form-group">
+                    <label for="description"><h3>Description</h3></label>
+                    <fieldset ng-disabled="isSubmitting">
+                        <textArea 
+                            ng-model-options="{updateOn: 'blur'}"
+                            ng-model="newPosting.description" 
+                            type="text" 
+                            class="form-control" 
+                            placeholder="Describe your post here..."                    
+                            id="postDescription"
+                            rows="4">
+                        </textArea>
+                    </fieldset>
+                    <div ng-repeat="message in descriptionErrorMessages">
+                        <h4 style="color: red">{{message}}</h4>
                     </div>
-                    <div class="text-center">
-                        <br/>
-                        <button type="submit" class="btn-block btn-primary">Save</button>
-                        <button type="submit" class="btn-block btn-default">Cancel</button>
-                        <br/>
-                        <br/>
+                </div>
+                <div class="form-group">
+                    <label><h3>Category</h3></label>
+                    <fieldset ng-disabled="isSubmitting">
+                        <select 
+                            ng-model="categoryDescription" 
+                            ng-options="category as category for category in categoryOptions"
```

Comment:
Looks weird for sure... I followed this S.O. post's advice: http://stackoverflow.com/questions/18202106/ng-options-with-simple-array-init